iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -66,7 +66,11 @@ public enum ErrorCode
         DeviceNotFound = 404001,
 
         JobNotFound = 404002,
-        PartitionNotFound = 404003,
```

Comment:
Take note we had the wrong error code in the first place...

Reply:
The new name for 404003 is CannotModifyImmutableConfigurationContent, in the service file. I don't see it being returned anywhere also, so not adding that code here seems right.